design/FY2023/NFP-DynamicCoatingWallRoof.md

```diff
@@ -0,0 +1,99 @@
+
+
+<p style="text-align: center;"> Table 1. Common stimuli of dynamic coating material.</p>
+
+|type|stimuli|Example studies|
+|----|-------|---------------|
+|Passive|Surface temperature|[1]–[4]|
+|Active	|Mechanical structure|[5], [6]|
+||Force (stretching or compression)|[7], [8]|
+||Distribution of chemical content|[9]|
+||Electric current|[10]|
+
+Based on the above brief review of dynamic coating material, we propose to enable the modeling of the stimuli-solar/thermal absorption in EnergyPlus directly as opposed to relying on EMS. We propose to add a new Material object to hold stimuli-solar/thermal absorption curve or table data. This user input will overwrite the corresponding material properties based on the value of the stimuli.
+
+## Approach
+
+A new idd object, *Material:DynamicCoating* will be added to host user input data on the stimuli-coating property relationship.
+
+## Testing/Validation/Data Source(s)
+
+The feature will be tested and demonstrated with a test file derived from *"RefBldgSmallOfficeNew2004_Chicago.idf* with various dynamic coatings. The simulation results will be verified against the EMS approach. One of the dynamic coating material to be simulated is from Professor Junqial Wu’s group at UC Berkeley [11]. The following is a plot of its surface temperature-emissivity relationship.
+
+![temperature_emissivity](temperature_emissivity.png)
+<p style="text-align: center;"> Figure 1. Surface temperature-emissivity relationship for the silver Temperature Adaptive Radiative Cooling material from Professor Junqiao Wu’s group [11].</p>
+
+## IDD Object changes
+
+A new object Material:DynamicCoating object is proposed.
+ 
+    Material:DynamicCoating,
+        \memo Materials with variable properties of thermal emissivity and/or solar reflectance
+        \min-fields 8
+      A1 , \field Name
+          \required-field
+          \type alpha
+          \reference MaterialName
+      …
+      N5 , \field Visible Absorptance
```

Comment:
Referring to @JasonGlazer 's comment. Does the `...` imply additional fields here? If so, please include all proposed fields. If not, then this can't be a complete material layer, because it won't have any thermal properties for the CTF or FD calculations. If not, then delete the `...`.

Reply:
Yes. I will add those fields in.

Reply:
Just added the in.